cinch/playbooks/jenkins_restart_utility.yml

```diff
@@ -5,9 +5,16 @@
 
 # Example:
 # ansible-playbook -u root --private-key /path/to/key
-# -i '10.0.183.169', jenkins_restart_utility.yml -e "restart_jenkins=true"
+# jenkins_restart_utility.yml
+# -e "restart_jenkins=true hostname=jenkins
+# domain=example.com"
+
+# Optionally include an arbitary role here
+# to run before the playbook is executed if needed"
+- hosts: "{{ hostname }}.{{ domain }}"
```

Comment:
I'm not so keen on forcing a host to be dynamically constructed out of variables, here. Why are you doing this instead of just referring to a group name, such as "run_pre_role" and letting inventory magic take care of things directly?

Reply:
One reason that comes to mind is that we need this to populate this dynamically from a ticketing system sending parameters via API to Ansible Tower. If we were to use inventories, we would need to include another wrapping playbook with a task which would use the add_host module to populate a proper inventory. I doubt this is wanted for upstream too and this is definitely an easier way to achieve this while not diverging from upstream.

Reply:
Can Tower not construct an inventory out of the argument?

Reply:
I have changed the behavior to use the add_host for constructing in-memory inventory, that should look nicer.